.github/workflows/pullrequests.yml

```diff
@@ -17,6 +17,14 @@ jobs:
         - 3.7
         - 3.8
         libraries:
+        - marshmallow=='3.0.*' flask=='1.0.*' werkzeug=='1.*'
+        - marshmallow=='3.1.*' flask=='1.1.*' werkzeug=='1.*'
+        - marshmallow=='3.2.*' flask=='1.1.*' werkzeug=='1.*'
+        - marshmallow=='3.3.*' flask=='1.1.*' werkzeug=='1.*'
+        - marshmallow=='3.4.*' flask=='1.1.*' werkzeug=='1.*'
+        - marshmallow=='3.5.*' flask=='1.1.*' werkzeug=='1.*'
+        - marshmallow=='3.6.*' flask=='1.1.*' werkzeug=='1.*'
+        - marshmallow=='3.7.*' flask=='1.1.*' werkzeug=='1.*'
```

Comment:
If we are dropping marshmallow v2, should the below lines running tests with marshmallow 2.x be dropped?